api/alerts.go

```diff
@@ -3,25 +3,31 @@ package api
 import (
 	"fmt"
 	graphql "github.com/cli/shurcooL-graphql"
+	"github.com/humio/cli/api/internal/humiographql"
 )
 
 type Alert struct {
-	ID                 string   `graphql:"id"                 yaml:"-"                     json:"id"`
-	Name               string   `graphql:"name"               yaml:"name"                  json:"name"`
-	QueryString        string   `graphql:"queryString"        yaml:"queryString"           json:"queryString"`
-	QueryStart         string   `graphql:"queryStart"         yaml:"queryStart"            json:"queryStart"`
-	ThrottleField      string   `graphql:"throttleField"      yaml:"throttleField"         json:"throttleField"`
-	TimeOfLastTrigger  int      `graphql:"timeOfLastTrigger"  yaml:"timeOfLastTrigger"     json:"timeOfLastTrigger"`
-	IsStarred          bool     `graphql:"isStarred"          yaml:"isStarred"             json:"isStarred"`
-	Description        string   `graphql:"description"        yaml:"description,omitempty" json:"description"`
-	ThrottleTimeMillis int      `graphql:"throttleTimeMillis" yaml:"throttleTimeMillis"    json:"throttleTimeMillis"`
-	Enabled            bool     `graphql:"enabled"            yaml:"enabled"               json:"enabled"`
-	Actions            []string `graphql:"actions"            yaml:"actions"               json:"actions"`
-	Labels             []string `graphql:"labels"             yaml:"labels,omitempty"      json:"labels,omitempty"`
-	LastError          string   `graphql:"lastError"          yaml:"lastError"             json:"lastError"`
+	ID                 string   `yaml:"-"`
+	Name               string   `yaml:"name"`
+	QueryString        string   `yaml:"queryString"`
+	QueryStart         string   `yaml:"queryStart"`
+	ThrottleField      string   `yaml:"throttleField"`
+	TimeOfLastTrigger  int      `yaml:"timeOfLastTrigger"`
+	IsStarred          bool     `yaml:"isStarred"`
+	Description        string   `yaml:"description"`
+	ThrottleTimeMillis int      `yaml:"throttleTimeMillis"`
+	Enabled            bool     `yaml:"enabled"`
+	Actions            []string `yaml:"actions"`
+	Labels             []string `yaml:"labels,omitempty"`
+	LastError          string   `yaml:"lastError"`
+	RunAsUserID        string   `yaml:"runAsUserId,omitempty"`
+	QueryOwnershipType string   `yaml:"queryOwnershipType,omitempty"`
```

Comment:
Not sure here, if removing the struct tags for `json` and `graphql` could be considered a breaking change.

There might be some consumer of this public API that is using `json.Marshal` elsewhere, which would now see a different output. Or a consumer that uses it for GraphQL 🤷 

This is really the problem with using the same data type in multiple places.

Reply:
That makes sense. Actually thought about it while falling asleep yesterday 😴 Reverted